section1/relection.md

```diff
@@ -2,18 +2,59 @@
+
 3. How would you log the string `"Hello World!"` to the console?
 
+var greeting = "Hello World!";
+console.log(greeting)
+
 4. What is/are the character(s) you would use to indicate comments in a JavaScript file? What is the purpose of a code comment?
 
+// The purpose of a code comment is to give insight to the purpose of the code.
+Code comments are sort of like an invisible note only we can see.
+Code comments detail the inputs of a code (info the code requires), and the outputs (anything created or returned from the code).
+
 5. In your own words, what is a variable? How would you explain it to a 5 year old?
 
+A variable is like a storage location paired with an associated value. I would use a chocolate chip cookie to explain a variable to a 5 year old. I would ask them to tell me everything they would say to describe the cookie. They would probably say something along the lines of "there is one cookie, it is brown, there are 5 chocolate chips, it is yummy." I could then tell them that each of those ways they describe a cookie could be considered a variable.
+
+For example:
+
+var name = "Chocolate chip cookie",
+var quantity = 1,
+var numberOfChocolateChips = 5,
+var color = "brown";
+var isYummy = true;
+
 6. Think of a site or app you use frequently. What are three variables that are probably used? Which data type would each of those variables probably hold?
 
+I just used Trip Action to book a work trip. Three variables they used are:
+
+String: "Welcome to Trip Action!" as their greeting.
+Boolean: To determine the reason for my reservations, they asked if this was a work trip. I had the option to click either Yes or No (aka true or false).
+
 7. In your own words, explain what concatenation is.
 
+Concatenation means joining together. Using concatenation, we are able to join strings together by using the "+" character.
+
 8. Think of a site or app you use frequently. Where do you think the developers used concatention?
 
-9. What questions do you still have about the work we've done so far? (not having a question is not an option)
\ No newline at end of file
+I just ordered flowers to be delivered to my mom on her birthday. When I selected the flowers and went to the checkout page, there was a message that said something like "You are ordering 1 item, the Birthday Flowers, being sent to mom." This was joining together multiple string variables to make a statement that was true.
+
+9. What questions do you still have about the work we've done so far? (not having a question is not an option)
+
+I am wondering how we would store a phone number as a variable and what data type that would be  considered. A phone number is a list of numbers, so my gut says it would be numeric. However, the more I think about it, the more I think it would be a string data type, because to return a phone number to a user, we would want the phone number to print as, for example, (555) 555-5555, not 5555555555.
```

Comment:
You're right! A phone number would be a string, because we would never perform any mathematical operations on a phone number. Same thing with zip codes and credit card numbers. Great thinking!